woocommerce/woocommerce-android
# [Hack week] Create a new product using images

Fixes #9281. This PR adds the ability to share images from different apps and launch the Woo app, which automatically creates a new product using the shared images. The feature was requested by an existing user.

https://github.com/woocommerce/woocommerce-android/assets/1522856/9e1f3446-b7bd-4b0f-bab1-6e2336c4d74b

**To test:**
1. Open an app that can share an image (like a gallery app)
2. Select one ore more images and share them
3. Select the WooCommerce app from the share dialog
4. Notice the WooCommerce app is launched and the selected images are automatically added to a new product and uploaded
